app/services/autostart-tour.js

```diff
@@ -1,5 +1,6 @@
 import Service from "@ember/service";
 import { tracked } from "@glimmer/tracking";
+import Tours from "timed/tours";
```

Comment:
```suggestion

import TOURS from "timed/tours";
```
As it is used like a constant we should mark it like that.